### PR TITLE
major changes

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by helish88 — COMMENTED** [review]
I think this variables are  useless
```py
giveawaychannel = None

since = None

msg4 = None

users = None```

File: main.py [hunk 1]
```diff
@@ -48,81 +39,81 @@ def check(m):
...
+    guild = client.get_guild(796081913688883240)  # Put your guild ID here!
+    logchannel = guild.get_channel(799722523054833664)  # Put your channel, you would like to send giveaway logs to.
+    await logchannel.send(embed=logembed)
 
     futuredate = datetime.utcnow() + timedelta(seconds=timewait)
-    embed1 = discord.Embed(color = discord.Color(random.randint(0x000000, 0xFFFFFF)), title=f"🎉GIVEAWAY🎉\n`{msg4.content}`",timestamp = futuredate, description=f'React with 🎉 to enter!\nEnd Date: {futuredate.strftime("%a, %b %d, %Y %I:%M %p")}\nHosted by: {ctx.author.mention}')
-    
+    embed1 = discord.Embed(color=discord.Color(random.randint(0x000000, 0xFFFFFF)),
+                           title=f"🎉GIVEAWAY🎉\n`{msg4.content}`", timestamp=futuredate,
+                           description=f'React with 🎉 to enter!\nEnd Date: {futuredate.strftime("%a, %b %d, %Y %I:%M %p")}\nHosted by: {ctx.author.mention}')
```
Comment (helish88) [inline main.py:115]: and this is wrong. must be only timestamp = futuredate

**Review by helish88 — COMMENTED** [review]
read comment about embed timestamp

File: main.py (hunk 1 above)
Comment (ghost) [reply main.py:115]: wdym?
`     futuredate = datetime.utcnow() + timedelta(seconds=timewait) `
it written there

**Comment by ghost** [comment]
> 
> 
> I think this variables are useless
> 
> ```python
> giveawaychannel = None
> 
> since = None
> 
> msg4 = None
> 
> users = None```
> ```

well it fixes the issue 
`local variable referenced before assignment`

File: main.py (hunk 1 above)
Comment (helish88) [reply main.py:115]: I meant this
```py
 description=f'React with 🎉 to enter!\nEnd Date: {futuredate.strftime("%a, %b %d, %Y %I:%M %p")```
 people from different time zones will see the same time

**Comment by helish88** [comment]
> > I think this variables are useless
> > ```python
> > giveawaychannel = None
> > 
> > since = None
> > 
> > msg4 = None
> > 
> > users = None```
> > ```
> 
> well it fixes the issue
> `local variable referenced before assignment`

weird. ill check

**Pull request merged**